assets/survey_themes/fruity_twentythree/navbar/navbar.scss

```
@@ -24,20 +24,19 @@ Responsive navbar-brand image CSS
 ***********************************/
 
 .logo-container {
-    padding: 0px;
+    padding: 0;
     float: left;
-
+    flex-shrink: 0;
+    
     font-size: 18px;
```

Stylelint (17.9.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)